_posts/en/newsletters/2021-01-06-newsletter.md

```diff
@@ -0,0 +1,139 @@
+  [posted][poelstra post] a partial rewrite of the [BIP322][]
+  generic message signing proposal.  The rewrite mainly clarifies the
+  signing and verification procedures, but it now also allows wallets
+  that don't implement a complete set of checks to return an
+  "inconclusive" state for signatures that use scripts they don't
+  understand.  For example, a developer who wants to quickly implement
+  generic signmessage can only implement signature verification for
+  P2PKH, P2WPKH, and P2SH-P2WPKH scripts, which covers a large
+  percentage of all current and historic outputs.  For anything else,
+  the wallet can return "inconclusive" and maybe direct the user to a
+  more comprehensive verification tool.
+
+- **Trampoline routing:** Bastien Teinturier [posted][teinturier post]
+  to the Lightning-Dev mailing list a new [specification][tramp spec]
+  for [trampoline routing][topic trampoline payments] which allows a
+  spender to pay a receiver without calculating a route between their
+  nodes.  Instead, the spender routes their money to a nearby trampoline
+  node who calculates the next part of the route---getting the money
+  either to the receiver or to the next trampoline node.  This is useful
+  for spenders with lightweight LN clients that are often offline or
+  otherwise unable to keep track of network gossip in order to calculate
+  routes themselves.
+
+    Teinturier notes that ACINQ has been running a single-bounce
+    trampoline node for users of their Phoenix wallet.  This provides
+    the route-finding benefit but, as described in a [blog post][acinq
+    post], it means ACINQ can see the destination of all its users'
+    payments.  The solution to this problem is allowing routing nodes
+    to advertise the ability to accept onion-wrapped trampoline routes
+    where spender Alice sends her money first to one trampoline (e.g.
+    ACINQ's node), then possibly to other trampolines, and finally to
+    the receiver node.  This prevents any individual trampoline from
+    learning the node identifier for both the spender and receiver.  The
+    proposed specification would provide a standardized way to achieve
+    this.
+
+    It's Teinturier opinion that "the code changes are very reasonable
+    as it reuses a lot of components that are already part of every
+    lightning implementation and doesn't introduce new assumptions."
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Eclair 0.5.0][] is a new major version that adds support for a
+  scalable cluster mode (see [Newsletter #128][news128 akka]), block
+  chain watchdogs ([#123][news123 watchdog]), and additional plugin
+  hooks, among many other features and bug fixes.
+
+- [Bitcoin Core 0.21.0rc4][Bitcoin Core 0.21.0] is a release candidate
```

Comment:
Is this list usually in alphabetical order? (e.g. Bitcoin Core before Eclair), ignore me if not

Reply:
Primary order is releases before RCs, which is why the Eclair release sorts before the Bitcoin Core RC.  Secondary order is the order of projects in the intro paragraph to the *Notable changes* section (which is my opinion on order of importance by type of project (full nodes > LN nodes > libraries > tools > documentation) and alphabetical within project type).

Reply:
Thanks! Noting, since I've probably asked this before.